Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+ruby RUBY_VERSION
+
+gem 'bundler'
```

Comment:
@kenichi **Update**: Have now tested without duplication of what was in the gemspec in the Gemfile, and it worked fine!